packages/components/time-picker/src/utils.ts

```diff
@@ -66,6 +67,8 @@ export const parseDate = function (
   const day =
     isEmpty(format) || format === 'x'
       ? dayjs(date).locale(lang)
+      : ['ww', 'w'].some((item) => format?.includes(item))
+      ? dayjs().week(replace(date as string, /[^0-9]/gi, '') as any)
```

Comment:
Why to use `replace` from `lodash`? Instead of `String#replace`?

Reply:
It'has different with both of them in use?